gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -236,25 +269,43 @@ void
 Dump::visit (BlockExpr &block_expr)
 {
   stream << "BlockExpr: [";
+
   indentation.increment ();
-  stream << std::endl;
   // TODO: inner attributes
+  if (!block_expr.inner_attrs.empty ())
+    {
+      stream << "\n";
+      stream << indentation << "inner_attrs: [";
```

Comment:
The '[' seems not to be terminated.

Reply:
> Almost there :) Style and structure is good. It would be best to compile some examples and check the output (and post it here).

I will try. But I will finish my homework first 😭.

Reply:
No worries. I am just procrastinating my homework by giving this review :lol: